### PR TITLE
Fix several bugs...

### PR DESCRIPTION
**Pull request by jtferns**
#### Stories
- https://www.pivotaltracker.com/story/show/66683682
  - The measure logic should not be collapsed by default
- https://www.pivotaltracker.com/story/show/66683740
  - Expand measure logic on change of selected patient
- https://www.pivotaltracker.com/story/show/66684144
  - re-add failing patients to coverage
- https://www.pivotaltracker.com/story/show/66684206
  - question mark icon should be replaced with N/A in text muted class
- https://www.pivotaltracker.com/story/show/66684130
  - Hide the "show" button when currently disabled
- https://www.pivotaltracker.com/story/show/66684836
  - create a white/black list link for portfolio user
- https://www.pivotaltracker.com/story/show/66684318
  - Align the show and add patient button and the patient names with PASS/FAIL
#### Screenshots
- measure logic should not be collapsed by default
  - one
- question mark icon should be replaced with N/A
  - two
- hide show coverage button when patient result is collapsed
  - three
- create link for portfolio user to white/black-list builder
  - four
- align show coverage button with add patient button and fix vertical alignment in patient result panel headings
  - five_2

### CODE REVIEW

File: app/assets/javascripts/templates/logic/population_criteria.hbs [hunk 1]
```diff
@@ -4,12 +4,12 @@
...
         <div>
           <span class="{{population.type}} rationale-target"><b>{{translate_population population.type}}: </b></span>
-          <div class="pull-right"><i class="fa fa-lg fa-angle-right toggle-icon"></i></div>
+          <div class="pull-right"><i class="fa fa-lg fa-angle-down toggle-icon"></i></div>
         </div> 
       </a>
     </h4>
   </div>
-  <div id="collapse{{cid}}" class="panel-collapse collapse">
+  <div id="collapse{{cid}}" class="panel-collapse collapse in">
```
Comment (carpeliam) [inline app/assets/javascripts/templates/logic/population_criteria.hbs:15]: I think you can actually accomplish the same thing here with the following: `<div id="collapse{{cid}}" class="panel-collapse">`

Comment (jtferns) [inline app/assets/javascripts/templates/logic/population_criteria.hbs:15]: `<div id="collapse{{cid}} class="panel-collapse">` expands out the population logic from the panel heading, even though it is already expanded...

`<div id="collapse{{cid}} class="panel-collapse in">` might be the desired initial class for the population panels?

Comment (carpeliam) [inline app/assets/javascripts/templates/logic/population_criteria.hbs:15]: Yep! you're right.

**Comment by carpeliam** [comment]
Everything looks ok at a glance, but it's hard to evaluate several issues in a single pull request (hard to look through and see what's related to what). Does anybody else want to take a peek? Or maybe it'd be better to go through them in person...?

**Comment by andrequina** [comment]
Lets get this done today if possible.

**Comment by carpeliam** [comment]
During standup, we talked about this; I've looked through the code and things look ok, but haven't tested each fix individually; @pkmitre mentioned he'd run through the fixes in a browser. We also resolved to keep PRs limited to individual features as they're easy to inspect and should get accepted more quickly.

A bit less important, but not totally unimportant, is that this style of PR also makes it almost impossible to adhere to our git workflow, as there are a number of branches merged into one, and they weren't committed sequentially.

**Pull request merged**